Sources/TuistKit/Services/EditService.swift

```diff
@@ -52,6 +52,10 @@ final class EditService {
     ) throws {
         let path = self.path(path)
 
+        let pathsToExclude = [
+            "**/\(Constants.tuistDirectoryName)/\(Constants.DependenciesDirectory.name)/**",
```

Comment:
As I mentioned in another comment, I don't think this has to be passed by the service. It can be part of the editing implementation.